### ISSUE TITLE
can't predict test

### ISSUE THREAD

**Issue by hoangnhancs** [ptran1203/traffic_sign_detection#10]
hello Phat, I run file train.py, and i get my new weights, but i can't detect any sign. Can you help me in this problem? Thank you!!!

**Comment by ptran1203** [comment]
Let me check again. How many epochs did you run?

**Comment by hoangnhancs** [comment]
i run with 10 epoch, and loss = 0.2 . can i connect with you on other communication

**Comment by ptran1203** [comment]
Yes, my Zalo: 0888104172
